BayPassTests/Model/Map/RouteSegmentTests.swift

```diff
@@ -45,9 +45,7 @@ class RouteSegmentTests: XCTestCase {
         assert(segment.arrivalTime == arrTime)
         assert(segment.polyline == polyline)
         assert(segment.travelMode == TravelMode.transit)
```

Comment:
You shouldn't just modify existing tests. If they fail you should look and fix your code, not the test case